app/facads/tmdb_search.rb

```diff
@@ -0,0 +1,11 @@
+class TmdbSearch
+
+  def service
+    TmdbService.new
```

Comment:
Services should have class methods so you don't need to instantiate them with `.new`